client/components/jetpack-header/index.jsx

```diff
@@ -23,11 +23,29 @@ export class JetpackHeader extends PureComponent {
 	static propTypes = {
 		darkColorScheme: PropTypes.bool,
 		partnerSlug: PropTypes.string,
+		isWoo: PropTypes.bool,
 		width: PropTypes.number,
 	};
 
 	renderLogo() {
-		const { darkColorScheme, partnerSlug, width } = this.props;
+		const { darkColorScheme, partnerSlug, width, isWoo } = this.props;
+
+		if ( isWoo ) {
```

Comment:
Minor but I suppose one route here would be to add a conditional for setting `partnerSlug` to `woocommerce` based on if `isWoo` is truthy or not... then instead of having this conditional return block, we could just add another case statement below

Reply:
I originally went that route, but see the discussion here: https://github.com/Automattic/wp-calypso/pull/33073#discussion_r284695767

Reply:
Aye, thanks for the background!